browser/plugins/three_meshline_mesh.plugin.js

```diff
@@ -0,0 +1,55 @@
+(function() {
+	var createThreeMeshRoot = function() {
+		return new THREE.Object3D()
+	}
+
+	var createThreeMeshLineMesh = function(geom, mats) {
+		// Have to convert the bufferGeometry to THREE.Geometry(), as the MeshLine only
+		// accepts THREE.Geometry()
+		var set_geom;
```

Comment:
would be good to be consistent with naming conventions regards camelCasing, i.e. setGeom